js/client-teambuilder.js

```diff
@@ -112,6 +112,12 @@
 		curSet: null,
 		curSetLoc: 0,
 
+		// This is used in the search bar in attempt of figuring
```

Comment:
"Instead of running a search after each character typed in the search bar we instead use a timer to only kick off a search 500ms after the last 'keyup' event (which is intended to approximate running a search only after the user has 'finished' typing)."